MarkdownView/src/us/feras/mdv/MarkdownView.java

```diff
@@ -74,10 +77,7 @@ private String readFileFromAsset(String url) throws IOException {
 		BufferedReader input = null;
 		StringBuilder contents = new StringBuilder();
 		try {
-			String assetFileName = url.substring(url.lastIndexOf('/') + 1,
-					url.length());
-			input = new BufferedReader(new InputStreamReader(getContext()
-					.getAssets().open(assetFileName)));
+			input = new BufferedReader(new InputStreamReader(getContext().getAssets().open(assetFilename)));
```

Comment:
I see that the assetFileName is not defined. What is the initialization for the variable supposed to be?